de.metas.contracts/src/main/java/de/metas/contracts/impl/FlatrateBL.java

```diff
@@ -1125,6 +1121,9 @@ private void extendContract0(
 			nextTerm.setDropShip_BPartner_ID(currentTerm.getDropShip_BPartner_ID());
 			nextTerm.setDropShip_Location_ID(currentTerm.getDropShip_Location_ID());
 
+			nextTerm.setC_TaxCategory_ID(currentTerm.getC_TaxCategory_ID());
```

Comment:
hi, i'm not 100% sure about the context, i only took a quick look, but: do you set the priceactual in here? i could not fid it..if it's not set in there, then i think you also should *not* set the two tax values.
i think it's computed somewhere in a listener..look for code calling the setter.
more: imho it should **not** just be copied. instead, it needs to be computed, just like priceactuall itself

Reply:
Yes, you are right! Fixed in follow up task: #2752